### PR TITLE
Update docs for CUDA10

### PR DESCRIPTION
**Pull request by ruimashita**
## Motivation and Context

related issue: #233

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,12 +24,12 @@ Check out the [Installation](https://docs.blue-oil.org/install/install.html) and
 
 ## Prerequisites
 - GNU/Linux x86_64 with kernel version > 3.10
-- NVIDIA GPU with Architecture > Fermi (2.1)
-- NVIDIA drivers >= 367.48
+- NVIDIA GPU with Architecture > 3.0 (Kepler)
```
Comment (kchygoe) [inline README.md]: =>

Comment (ruimashita) [reply README.md]: Thanks.

**Comment by ruimashita** [comment]
run blueoil test

**Comment by hadusam** [comment]
run blueoil test

**Pull request merged**